src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentManager.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Copilot;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.Language.Suggestions;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.DocumentationComments
+{
+    [Export(typeof(CopilotGenerateDocumentationCommentManager))]
+    internal class CopilotGenerateDocumentationCommentManager
+    {
+        private readonly SuggestionServiceBase _suggestionServiceBase;
+        private readonly IThreadingContext _threadingContext;
+        private readonly IAsynchronousOperationListener _asyncListener;
+
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public CopilotGenerateDocumentationCommentManager(SuggestionServiceBase suggestionServiceBase, IThreadingContext threadingContext,
```

Comment:
I think this is exported by intellicode, right? If so it might be null (even when hosted in VS). We should simply noop in that case